containers/daap-python-base/src/var/task/versioning.py

```diff
@@ -88,6 +91,28 @@ def __init__(self, data_product_name, logger: DataPlatformLogger):
         self.data_product_config = DataProductConfig(name=data_product_name)
         self.logger = logger
 
+    def remove_all_versions_of_data_product(self):
+        """Completely removes a data product, including tables, schemas, metadata
+        raw & curated data"""
+
+        # Delete the Glue Database
+        delete_glue_database(
+            data_product_name=self.data_product_config.name, logger=self.logger
+        )
+
+        # Delete raw files
```

Comment:
This should also delete the failed uploads from fail/ in the data bucket.

I think we should keep the files in the log bucket though.

Reply:
Completed